lib/api/apiUtils/object/versioning.js

```diff
@@ -55,14 +55,46 @@ function _storeNullVersionMD(bucketName, objKey, objMD, options, log, cb) {
     });
 }
 
+function _getVersionLocation(bucketName, objKey, options, log, cb) {
```

Comment:
I added a jsdoc and updated the name. It's only used for getting the delete locations.